evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -135,7 +135,8 @@ open class PackageRule(
      * A DSL function to configure a [LicenseRule] and add it to this rule.
      */
     fun licenseRule(name: String, licenseView: LicenseView, block: LicenseRule.() -> Unit) {
-        resolvedLicenseInfo.filter(licenseView, filterSources = true).forEach { resolvedLicense ->
+        resolvedLicenseInfo.filter(licenseView, filterSources = true)
+            .filter(ruleSet.ortResult.getLicenseChoices(pkg.id)).forEach { resolvedLicense ->
```

Comment:
Could this impact the performance of the evaluator for larger inputs?

Reply:
We don't think that this impacts the performance of the evaluator.
We renamed the function from `filter` to `applyLicense` as the functionality is different from the existing filters.

The peformance impact is basically to build the effective license and decompose it again which will lead to n additional loops for n Single SPDX licenses and if required to apply the choices.

Reply:
If its clear that there is not performance impact fine, otherwise it'd make sense IMO to run a quick test with a large project.